### DISCUSSION TITLE
Is there an easy way to append string to existing string like you do with number?

### DISCUSSION THREAD

**Discussion by logeeker** [prisma/prisma#13390] [Q&A] [answered]
Like for numbers you can do things like
```
const updatePosts = await prisma.post.updateMany({
  data: {
    views: {
      increment: 1,
    },
  },
})
```
What's the easiest way to append a string to an existing string? 

For example, if the existing `characters` field equals `abc`, how can I append `def` to the `characters` field so that it equals `abcdef` after the operation?

**Comment by nurul3101** [comment, marked as answer]
Hey @logeeker 👋 

I don't think there's any other easy way, you would need to first get the characters with [`findFirst`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findfirst) / [`findUnique`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findunique) and then [`update`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#update) it by appending your string.

**Comment by nmfrankel** [comment]
Upvote my [feature request](https://github.com/prisma/prisma/issues/15269) for this feature to be considered.
